src/telemetry.ts

```diff
@@ -422,3 +422,14 @@ function getPackageInfo(): IPackageInfo {
     aiKey: "AIF-d9b70cd4-b9f9-4d70-929b-a071c400b217",
   };
 }
+
+export type ConfigureOnOpenScope = "user" | "workspace";
+export function logConfigureOnOpenTelemetry(
+  configureOnOpen: boolean,
+  scope: ConfigureOnOpenScope | undefined = undefined
+) {
+  logEvent("configureOnOpenPopup", {
+    configureOnOpen: configureOnOpen.toString(),

```

Comment:
Would this emit "[object null]" if configureonOpenPopup is never assigned?

Reply:
I'm not sure what you mean? configureOnOpenPopup is simply the name of the event, so it will always emit an event and it will tag along those properties of `configureOnOpen` and `scope`.